content/en/blog/2023/announcing-notation-rc4.md

```diff
@@ -3,6 +3,8 @@ title: Notation v1.0.0-RC.4 is available!
 author:  "Notation Release Team"
 date: 2023-04-21
 draft: false
+image: "rc-latest.svg"
```

Comment:
This svg is not good, can we use the default one?

Reply:
> @kayceeDev The layout in my 13'inch laptop looks not comfortable. Is it possible to adjust its layout and size? ![image](https://user-images.githubusercontent.com/40452856/240907909-885bdf7a-4241-4068-bb18-791b918a6d81.png)

Are you referring to the vertical card or the horizontal one?